frontend/catalyst/jit.py

```diff
@@ -303,6 +323,11 @@ def _validate_configuration(self):
         if not hasattr(self.original_function, "__name__"):
             self.__name__ = "unknown"  # allow these cases anyways?
 
+        if not self.compile_options.autograph:
+            assert (
+                len(self.compile_options.autograph_include) == 0
+            ), "In order for 'autograph_include' to work, 'autograph' must be set to True"
```

Comment:
This should be a real exception since the user can trigger it with their input :)

Reply:
True. Done!